src/components/auth/Logout.jsx

```diff
@@ -3,9 +3,12 @@ import {Redirect} from 'react-router-dom';
 import {useDispatch, useSelector} from 'react-redux';
 import {useTranslation} from 'react-i18next';
 
-import colors from '../../stylesheets/config/colors.module.scss';
```

Comment:
I'm glad we use external/internal/style way with imports :)

Reply:
Unfortunately nothing has changed in this regard. This line has been moved below the other imports.